src/main/java/nl/knaw/dans/easy/DataciteService.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2014 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
- *
+ * <p>
```

Comment:
what's these `<p>` tags about?

Reply:
i don't know, unwanted gift from intellij. Removed since it blocks the license validator